### ISSUE TITLE
No need to atomically set VO bit in post_alloc?

### ISSUE THREAD

**Issue by qinsoon** [mmtk/mmtk-core#1156]
For allocators that have its allocation buffer, it seems unnecesary to set VO bit atomically.

https://github.com/mmtk/mmtk-core/blob/3be73b8048df17f1f2bd019d0b109488a2c5e313/src/policy/immix/immixspace.rs#L139-L142

https://github.com/mmtk/mmtk-core/blob/3be73b8048df17f1f2bd019d0b109488a2c5e313/src/util/metadata/vo_bit/mod.rs#L64-L71

**Comment by wks** [comment]
That's right.  It should include all bump-pointer allocators and the native MarkSweep, too (because each thread allocates in its own block).
